manifest.toml

```diff
@@ -0,0 +1,90 @@
+packaging_format = 2
+
+id = "dendrite"
+name = "Dendrite"
+description.en = "Matrix homeserver of second generation"
+description.fr = "Serveur Matrix de seconde génération"
+
+version = "0.12.0~ynh1"
+
+maintainers = ["tituspijean"]
+
+[upstream]
+license = "Apache-2.0"
+website = "https://matrix.org/"
+admindoc = "https://github.com/matrix-org/dendrite/tree/master/docs"
+code = "https://github.com/matrix-org/dendrite"
+
+[integration]
+yunohost = ">= 11.1.17"
+architectures = "all"
+multi_instance = false
+ldap = "not_relevant"
+sso = "not_relevant"
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.warning]
+    ask.en = "Dendrite is still in beta phase.\nIt is functional, but all Matrix features are yet to be implemented.\nSee the upstream repo for more information: https://github.com/matrix-org/dendrite\n\nThis package est encore en development, all features from Matrix/Synapse are yet to be implemented."
+    ask.fr = "Dendrite est encore en phase bêta.\nElle est fonctionnelle, mais n'implémente pas encore toutes les fonctionnalités de Matrix.\nConsultez le dépôt source pour plus d'information : https://github.com/matrix-org/dendrite\n\nCe package est encore en développement, toutes les fonctionalités de Matrix/Synapse ne sont pas encore implémentées."
+    type = "display_text"
```

Comment:
(Maybe to be replaced with a `PRE_INSTALL.md`)

Reply:
Solved by eric ❤️